code/index.html

```diff
@@ -1,17 +1,161 @@
+        <div class="hamburger-layer"></div>
+      </div>
+      <ul class="nav-menu">
+        <li><a href="#home">Home</a></li>
+        <li><a href="#the-swedes">The Swedes</a></li>
+        <li><a href="#vibes">Vibes</a></li>
+        <li><a href="#review">Review</a></li>
+      </ul>
+      <div class="social-media">
+        <a href="https://www.facebook.com/theroyalswedes" target="_blank">
+          <img src="./images/fb.png">
+        </a>
+        <a href="https://www.instagram.com/theroyalswedes/" target="_blank">
+          <img src="./images/insta.png">
+        </a>
+        <a href="https://www.youtube.com/channel/UCBAikymgmy7C5lQ5EhJT4bw/featured" target="_blank">
+          <img src="./images/youtube.png">
+        </a>
+        <a href="https://www.tiktok.com/@theroyalswedes" target="_blank">
+          <img src="./images/tiktok.png">
+        </a>
+      </div>
 
-  </body>
-</html>
+    </div>
+    <!-- 3D logo -->
+    <div class="logo-container">
+      <div class="logo-top">
+        <img src="./images/swedes-logo-top.jpeg" alt="logo">
+      </div>
+      <div class="logo-bottom">
+        <img src="./images/swedes-logo-bottom1.jpeg" alt="logo" class="logo">
+      </div>
+    </div>
+  </header>
+
+  <div class="image">
+    <img src="./images/the-royal-swedes-live-3.jpeg">
+  </div>
+
+  <div class="description" id="the-swedes">
+    <br>
+    <p>Hailing from the land of vikings and vibrant melodies, The Royal Swedes blend pop and rock with regal flair.
+      Featuring timeless
+      classics and contemporary hits, they captivate audiences with a sound that is both bold and nostalgic. <br><br>
+      Whether
+      it's
+      reliving the glory days of rock or creating new memories on the dance floor, The Royal Swedes know how to turn
+      any event into a celebration fit for royalty.</p><br>
+  </div>
+
+  <div class="grid-container" id="vibes">
+    <video class="video-bg" autoplay muted loop playsinline>
+      <source src="./video/royal-swedes-live - Made with Clipchamp.mp4" type="video/mp4">
+    </video>
+    <div class="grid-item">
+      <h2>"One of the best groups I have seen in a long time. Their music has a wide variety, they cater for all tastes,
+        and the lead singer has such a brilliant voice."</h2>
+      <p>Linda H.</p>
+    </div>
+    <div class="grid-item">
+      <h2>"The band is so talented and full of energy. They know how to entertain and interact wonderfully with the
+        crowd. 10 out of 10."</h2>
+      <p>Brenda K.</p>
+    </div>
+    <div class="grid-item">
+      <h2>"Kunde spela väldigt blandad musik. Superbra!!!"</h2>
+      <p>Marjo R.</p>
+    </div>
+    <div class="grid-item">
+      <h2>"Definitely worth attending their performance. Powerful, energetic and very talented."</h2>
+      <p>Charles G.</p>
+    </div>
+  </div>
+
+  <div class="form-container" id="review">
+    <div class="form">
+      <form action="https:httpbin.org/anything" method="post">
```

Comment:
This link is faulty, it should be: `https://httpbin.org/anything`

Reply:
OK, link should be correct now 🔗